docs/for-developers/index.md

```diff
@@ -0,0 +1,4 @@
+# For developers
+
+If you are a developer and would like to contribute to the REANA cluster source code,
```

Comment:
If you are a developer and would like to install REANA locally and contribute to the REANA cluster code, please see [REANA wiki on GitHub](https://github.com/reanahub/reana/wiki/Development-installation).